src/main/java/edu/ucsb/cs156/happiercows/entities/UserCommons.java

```diff
@@ -33,5 +33,14 @@ public class UserCommons {
   private int numOfCows;
 
   private double cowHealth;
+
+  private LocalDateTime startingDate;
+
+  private LocalDateTime lastDate;
+
+  public boolean gameInProgress(){
```

Comment:
Please move this function to the Commons.  It belongs there, not on the UserCommons.